### ISSUE TITLE
wk1 jul_epic: Update old data ui files on SharePoint Online for compliancy

### ISSUE THREAD

**Issue by lijenn** [lzim/teampsd#2049]
@lzim @staceypark Regarding the removal of the SSN column, we will also need to make the edits in the old data ui files on SharePoint Online. 

To reduce the rework of re-running the data and clinics with the updated Master Data UI, @lijenn and @anazariz believe we could delete the following on the old data ui files:
- [ ] remove Column G from the dataDiag and dataHF tabs
- [ ] remove the "Get Patient Level Data" and "Create Team Data Table" buttons to prevent the file from being re-run.

This was confirmed on 6/10 that the old data ui files on On-Prem will not override the changes above to be made on SharePoint Online.

**Comment by lijenn** [comment]
@lzim @staceypark 
Related to the file compliancy and migration for SharePoint Online, I believe we still need to do the following:

**1. Delete the SSN column and the “Get Patient Level Data” and “Create Team Data Table” buttons in the old data ui files on the new SharePoint Online.**
Based on the removal of the ssn column that Ash has fixed for the master data ui file, the column is still present in the old data ui files on the new SharePoint.
- @anazariz  and I have decided that the easiest fix which is to not re-run the old data and clinic selections with the updated master data ui file, but to manually delete the SSN column and the “Get Patient Level Data” and “Create Team Data Table” button in case someone does go into the old data ui files and re-run the data in the old files.
    - Anything we’re missing here?
- Also, I tried this on Jane and Matt's team's data ui file and the scan was still detecting the SSN issue.
When I manually click to resolve the scan, I get the pop-up below.
     - Should I continue? Unsure of who the report would get sent to, the admin of our SharePoint site?
![image](https://user-images.githubusercontent.com/59668647/121964671-c0123b80-cd20-11eb-80d2-6b63ae58432c.png)

**2. Message on On-Premise SharePoint regarding the migration/down-time.**
We will be mentioning about the migration to the Facilitators at tomorrow’s Facilitate Workgroup meeting. 
We should also add a message on the homepage of the On-Premise SharePoint as well for Learners or others that may be accessing the site.

**Could say/look something like this:
We are moving to SharePoint Online - The site will be down from this Friday 6/18 4pm Pacific/7pm Eastern until Monday 6/21 12am Eastern.**
